apps/website/components/ui/GlobalToast.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <v-snackbar
+    v-if="toast"
+    v-model="toast.show"
+    :color="getColor(toast?.type)"
+    location="top right"
+    timeout="5000"
+    @update:model-value="onUpdate"
+  >
+    <div class="d-flex align-center">
+      <v-icon :icon="getIcon(toast?.type)" class="mr-2" />
+      {{ toast?.message }}
+    </div>
+
+    <template #actions>
+      <v-btn icon="mdi-close" variant="text" @click="hideToast" />
+    </template>
+  </v-snackbar>
+</template>
+
+<script setup lang="ts">
+  import { useToast } from '~/composables/useToast';
+
+  /**
+   * Toast notification types supported by the component
+   */
+  export type ToastType = 'success' | 'error' | 'warning' | 'info';
+
+  /**
+   * Toast color mapping for Vuetify components
```

Comment:
Hi @IonutzHerbil , thanks for this! It would be nice to keep the consistent and use enum for toast type option and also place the file in the types/shared/ui.ts